code/game/objects/structures/signs/signs_interactive.dm

```diff
@@ -24,3 +24,50 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/calendar, 32)
 		. += span_info("Events:")
 		for(var/holidayname in SSevents.holidays)
 			. += span_info("[holidayname]")
+
+GLOBAL_LIST_EMPTY(map_delamination_counters)
+
+/obj/structure/sign/delamination_counter
+	name = "delamination counter"
+	desc = "A pair of flip signs describe how long it's been since the last delamination incident."
+	icon_state = "days_since_explosion"
+	var/since_last = 0
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/delamination_counter, 32)
+
+/obj/structure/sign/delamination_counter/Initialize(mapload)
+	. = ..()
+	if (mapload)
+		GLOB.map_delamination_counters += src
+	else
+		update_count(SSpersistence.rounds_since_engine_exploded)
```

Comment:
Ah, that's why it passed checks. damn mapload
I know the effect of this is nill, but could you clear out this list on deletion? sooths my neurotic brain